app/code/Magento/Quote/Test/Unit/Model/QuoteManagementTest.php

```diff
@@ -762,6 +793,16 @@ public function testPlaceOrder()
             ->method('setCustomerIsGuest')
             ->with(true);
 
+        $this->remoteAddressMock
+            ->expects($this->once())
+            ->method('getRemoteAddress')
+            ->willReturn($remoteAddress);
+
+        $this->requestMock
+            ->expects($this->once())
+            ->method('getServer')
+            ->willReturn($xForwarderForHeader);
```

Comment:
We must assert data was actually set to quote, please consider making a separate tiny test method. Don't use unneeded execution flow checks like `$this->once`, we prefer to verify code behavior rather than execution flow.

Reply:
ok, I will remove that, but in the same test case is `$this->once` already used several times for similar checks.
I already tried to create an own test method, but it's hard because I have to copy&paste a lot of stuff.